website/docs/middleware/logger.md

```diff
@@ -110,17 +117,87 @@ DefaultLoggerConfig = LoggerConfig{
 
 RequestLogger middleware allows developer fully to customize what is logged and how it is logged and is more suitable
 for usage with 3rd party (structured logging) libraries.
-
-See [`RequestLoggerConfig`](https://github.com/labstack/echo/blob/master/middleware/request_logger.go) structure fields for values that logger knows to extract.
```

Comment:
Please, leave that link in. Sometimes our docs get outdated and then it is useful for viewer to see quickly actual struct.